### PR TITLE
Feature/update to polkadot v0 9 17

### PR DESCRIPTION
**Pull request by MRamanenkau**
Pallets Referenda and Conviction votion are removed. The details are [here](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#c243f39b731d4246ade8aba0b7935bd9)

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 299,
+	spec_version: 300,
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:132]: Eee!

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -389,7 +390,7 @@ impl pallet_babe::Config for Runtime {
 }
 
 parameter_types! {
-	pub const IndexDeposit: Balance = 10 * DOLLARS;
+	pub const IndexDeposit: Balance = 1 * DOLLARS;
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs]: How can it affect us?

File: bin/node/runtime/src/lib.rs (hunk 1 above)
Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs:132]: Happy anniversary! :-)

File: bin/node/runtime/src/lib.rs (hunk 2 above)
Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs]: Wrong merge. Updated

**Review by AndreiNavoichyk — DISMISSED** [review]

File: bin/node/runtime/src/lib.rs (hunk 2 above)
Comment (AndreiNavoichyk) [reply bin/node/runtime/src/lib.rs]: Can we have such issues in other places?

Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs]: I chacked other constants after this mistake

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**